csrank/choicefunction/cmpnet_choice.py

```diff
@@ -170,6 +172,9 @@ def fit(
                 logger.info(
                     "Fitting utility function finished. Start tuning threshold."
                 )
+                self.threshold_ = self._tune_threshold(
```

Comment:
I think it was a bug that this was previously missing. Without this line, the threshold would always be `0.5` as set in `__init__`. The bug was revealed when I removed this default value from `__init__`. @kiudee am I right here?

Reply:
(All the other classes follow the pattern of using `_tune_threshold` here.)

Reply:
This is probably easier to see after #155 is done and the diff is cleaned up a bit.